docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+postgres:
+  image: postgres:9.3
+  environment:
+    POSTGRES_PASSWORD: mysecretpassword
+    POSTGRES_USER: postgres
+redis:
+  image: redis
+web:
+  build: .
+  volumes:
+    - ./:/opt/hummingbird/
+  ports:
+    - 3000
```

Comment:
Can this be changed to `3000:3000`? 

Need to know the port the API is running on to send AJAX requests from the browser.